docs/internal.md

```diff
@@ -118,6 +118,7 @@ The volume label is "cidata", as defined by [cloud-init NoCloud](https://cloudin
 - `LIMA_CIDATA_MNT`: the mount point of the disk. `/mnt/lima-cidata`.
 - `LIMA_CIDATA_USER`: the user name string
 - `LIMA_CIDATA_UID`: the numeric UID
+- `LIMA_CIDATA_HOME_MOUNTPOINT`: the mount point of `~`, or empty if not mounted
```

Comment:
```suggestion
- `LIMA_CIDATA_HOSTHOME_MOUNTPOINT`: the mount point of the host home directory, or empty if not mounted
```

Reply:
I thought it would be obvious that this is the host home, because the guest home is either `/home/${LIMA_CIDATA_USER}.linux` when you are `root`, or just `$HOME` when you are not.

But it doesn't hurt to be explicit, so I made the change.